documentation/antora.yml

```diff
@@ -20,7 +20,7 @@ asciidoc:
     apicurio-version: '2.2.5.Final'
     db2-version: '11.5.0.0'
     ojdbc8-version: '21.6.0.0'
-    community: true
+    community: '@'
```

Comment:
Hi @ahus1 I must admit, I'm not familiar with this syntax.  What exactly does `'@'` imply?  Is this in the documentation somewhere that explains this?

Reply:
The '@' at the end of the value implies a "soft set" attribute, see https://docs.antora.org/antora/latest/component-attributes/#soft-set for more information. The `@` is the marker for the "soft set", and will be stripped / will not be part of the value of the attribute.

This would allow you to override the attribute `community` in the `.asciidoctorconfig` file in a later step. Theoretically it would allow values to overwritten within a page as well, but that wasn't intended here. 

To be honest, this will not be needed unless one would want to overwrite the setting of `community` in the `.asciidoctorconfig` file. 

I could remove this from the PR, and it could be part of a future PR when there is a more obvious need for it. 

WDYT?

Reply:
I removed the changes to `antora.yml` in the PR.

Reply:
Hi @ahus1, great, thanks for the explanation, I agree that leaving `antora.yml` as is, for now, makes the most sense given that it doesn't have any impact.  If we find that Bob wants to use this toggle we can think about adding it back.